up42/order.py

```diff
@@ -118,21 +133,40 @@ def place(cls, auth: Auth, order_parameters: dict) -> "Order":
         return order
 
     @staticmethod
-    def estimate(auth: Auth, order_parameters: dict) -> int:
+    def estimate(auth: Auth, order_parameters: OrderParams) -> int:
         """
         Returns an estimation of the cost of an order.
 
         Args:
             auth: An authentication object.
-            order_parameters: A dictionary like {dataProduct: ..., "params": {"id": ..., "aoi": ...}}
+            order_parameters: A dictionary for the order configuration of the type OrderParams.
 
         Returns:
             int: The estimated cost of the order
         """
-        url = endpoint(f"/workspaces/{auth.workspace_id}/orders/estimate")
 
-        response_json = auth._request(request_type="POST", url=url, data=order_parameters)
-        estimated_credits: int = response_json["data"]["credits"]  # type: ignore
+        def translate_construct_parameters(order_parameters):
+            feature_collection_template = {"type": "FeatureCollection", "features": []}
+            order_parameters_v2 = deepcopy(order_parameters)
+            params = order_parameters_v2.get("params", {})
+            order_parameters_v2["displayName"] = params.get("id", "Display Name")
+            feature = params.pop("aoi", None)
+            if feature:
```

Comment:
Would estimation even work without aoi?

Reply:
There are some providers that do not have aoi as they are full scene. Therefore, there should be some cases with no aoi.